pyproject.toml

```diff
@@ -16,7 +16,13 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 
-dependencies = ["pydantic==1.8.2"]
+dependencies = [
+  "pydantic>=2.4.2", # dolma does not work with very old versions of pydantic
+  "dolma@git+https://github.com/allenai/dolma.git", # Install from git until a 0.9.2 package is released
+  "kenlm", # Used for perplexity tagging
+  "blingfire", # Used for perplexity tagging
+  "requests",
```

Comment:
Set a minimum version on these

Reply:
Perhaps a pinned version instead? Since DFM is more of an application than a library, and pinned versions maximise reproducibility :-)

Reply:
I see your point. I prefer minimum versions because it's much less strict and I think we should try to be compatible with the latest releases of libraries as long as the DFM project is maintained.